tests/js/zamboni/stats.spec.js

```diff
@@ -124,4 +124,110 @@ describe(__filename, () => {
       });
     });
   });
+
+  describe('stats/overview.js', () => {
+    let stats_overview_make_handler;
+
+    beforeEach(() => {
+      const stats_all = require('../../../static/js/zamboni/stats-all.js');
+
+      stats_overview_make_handler = stats_all.stats_overview_make_handler;
+    });
+
+    describe('"in-range" dates', () => {
+      const createMinimalHTML = () => {
+        return `
+<div class="primary" data-report="overview">
+  <div id="downloads-in-range"></div>
+  <div id="users-in-range"></div>
+</div>`;
+      };
+
+      beforeEach(() => {
+        const date = new Date(2019, 10 - 1, 14);
```

Comment:
Why do we need this date setup? It doesn't seem like any of the tests care about this date.